open-metadata-implementation/view-services/server-author-view/server-author-view-server/src/main/java/org/odpi/openmetadata/viewservices/serverauthor/handlers/ServerAuthorViewHandler.java

```diff
@@ -0,0 +1,623 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.viewservices.serverauthor.handlers;
+
+import org.odpi.openmetadata.adminservices.client.MetadataAccessPointConfigurationClient;
+import org.odpi.openmetadata.adminservices.client.MetadataServerConfigurationClient;
+import org.odpi.openmetadata.adminservices.client.OMAGServerConfigurationClient;
+import org.odpi.openmetadata.adminservices.configuration.properties.EnterpriseAccessConfig;
+import org.odpi.openmetadata.adminservices.configuration.properties.OMAGServerConfig;
+import org.odpi.openmetadata.adminservices.configuration.properties.ResourceEndpointConfig;
+import org.odpi.openmetadata.adminservices.ffdc.exception.OMAGConfigurationErrorException;
+import org.odpi.openmetadata.adminservices.ffdc.exception.OMAGInvalidParameterException;
+import org.odpi.openmetadata.adminservices.ffdc.exception.OMAGNotAuthorizedException;
+import org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.Connection;
+import org.odpi.openmetadata.viewservices.serverauthor.api.ffdc.ServerAuthorExceptionHandler;
+import org.odpi.openmetadata.viewservices.serverauthor.api.ffdc.ServerAuthorViewErrorCode;
+import org.odpi.openmetadata.viewservices.serverauthor.api.ffdc.ServerAuthorViewServiceException;
+import org.odpi.openmetadata.viewservices.serverauthor.api.properties.ResourceEndpoint;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * The TexViewHandler is initialised with the server the call should be sent to.
+ * The handler exposes methods for functionality for the type explorer view
+ */
+public class ServerAuthorViewHandler {
+//    private static final Logger log = LoggerFactory.getLogger(ServerAuthorViewHandler.class);
+
+    /*
+     * Specify a constant for the (max) length to which labels will be truncated.
+     */
+    private static final int TRUNCATED_STRING_LENGTH = 24;
+
+
+    /*
+    * viewServiceOptions should have been validated in the Admin layer.
+    * The viewServiceOptions contains a list of resource endpoints that the
+    * view service can connect to. It is formatted like this:
+    * "resourceEndpoints" : [
+                {
```

Comment:
This is actually an out of date config example. Sorry that's my fault - there is a current one now in Rex and the other Integration View Services - could you please pick that one up?